### PR TITLE
locators refactor + added new simple case login as user

### PR DESCRIPTION
**Pull request by jstawow**

### CODE REVIEW

File: src/test/functional/pages/LoginPage.ts [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+  }
 
     async loginUser(mail: string, password: string): Promise<void> {
-        await this.page.type(loginPageLocator.nameInput, mail);
-        await this.page.type(loginPageLocator.passwordInput, password);
-        await this.page.click(loginPageLocator.loginButton);
+        await this.userNameInput.type(mail);
+        await this.passwordInput.type( password);
+        await this.loginButton.click();
       }
```
Comment (annatooploox) [inline src/test/functional/pages/LoginPage.ts:23]: @jstawow dodałabym tu asercje i sprawdziła czy dobrym user się zalogowała. By było konsekwentnie, jak w drugim TC

**Review by annatooploox — APPROVED** [review]

File: src/test/functional/tests/loginPage.spec.ts [hunk 2]
```diff
@@ -16,21 +14,17 @@ test.describe("Login Page", () => {
...
-
-
-
-
-
-
-
-
+  test("Test Case 2: Login As A User", async ({ page }) => {
+    const loginPage = new LoginPage(page);
```
Comment (piotrw91) [inline src/test/functional/tests/loginPage.spec.ts]: You create login page in the beforeEach function so I guess we can avoid creating it again in each test?

Comment (jstawow) [reply src/test/functional/tests/loginPage.spec.ts]: yes i know but maybe in future we will skip logging in beforeEach?

File: src/test/functional/data.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 export const userTestData = {
```
Comment (piotrw91) [inline src/test/functional/data.ts]: ### This is just a suggestion for easier maintenance and readability in the future.
You have a const here named userTestData - which indicates that this dictionary stores normal user data? But after that we can see that there is also admin data in the dictionary. My suggestion if you would like to do it like this - split it into two separate dictonaries. First for normal user and second for admin user. After that you can call the whole user dictionary in tests. Something like this?:
```
export const normalUserTestData = {
  password: "passwordNormalUser",
  userName: "Normal User"
};

export const adminUserTestData = {
  password: "passwordAdminUser",
  userName: "Admin User"
};
```
And then in test you can call it as single parameter not two.
`await loginPage.loginUser(testData.adminUserTestData)`

Comment (jstawow) [reply src/test/functional/data.ts]: right i will improve it

**Review by piotrw91 — APPROVED** [review]

**Pull request merged**